src/en/charms-fan-aws-vpc.md

```diff
@@ -0,0 +1,65 @@
+Title: Creating an AWS VPC
+
+# Creating an AWS VPC
+
```

Comment:
It's worth noting that, as of today, a default VPC will be created automatically with users' account.

Reply:
I mentioned it on the main page but I can repeat here I suppose.